### PR TITLE
Proposed amendment on html/css file

### PR DESCRIPTION
**Pull request by zeff96**
Add HTML, CSS, readme, and license to the document.

I have created the HTML page on this branch, and styled all its children with the relevant CSS properties, equally I have updated the readme file to a professional level and added a license to the document.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+</head>
+<body>
+  <header class="header">
+    <nav class="top-nav">
+      <div class="logo">
+        <p>
+          my logo
```
Comment (tufoinnkuo10) [inline index.html]: - [x] kindly ensure that your logo has a link (a tag).

This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Personal-Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+</head>
+<body>
+  <header class="header">
+    <nav class="top-nav">
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] i can see you have two nav's . please kindly use just the nav for the hamburger icon. The `nav` tag is for semantics reason

File: index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Personal-Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+</head>
+<body>
+  <header class="header">
```
Comment (tufoinnkuo10) [inline index.html:14]: - [ ] please kindly ensure that the header tag covers just the tool bar section. the headline section will be covered by the `section tag`. the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      <nav class="nav-items">
+        <div class="hamburger-menu">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+      </nav>
+    </nav>
+    <section class="text">
+      <header class="text-section">
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] kindly use a div tag here instead of a header tag.a header tag is used as a page header

File: index.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        <div class="social-media">
+          <p class="support-text">
+            LET&#8217;S CONNECT
+          </p>
+          <div class="social-media-icons">
+            <a href=""><i class="fa-brands fa-twitter fa-2x twitter"></i></a>
+            <a href=""><i class="fa-brands fa-linkedin fa-2x linkedin"></i></a>
+            <a href=""><i class="fa-brands fa-medium fa-2x medium"></i></a>
+            <a href=""><i class="fa-brands fa-github fa-2x github"></i></a>
+            <a href=""><i class="fa-brands fa-angellist fa-2x angellist"></i></a>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please kindly ensure that The list of icons uses a `ul` tag to group them, and `li` per icon;

File: index.html [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+      </nav>
+    </nav>
+    <section class="text">
+      <header class="text-section">
+        <h1 class="head-section">
+          I&#8217;m Adeka Glad to see you!
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] nice job. please kindly add a background image not only to make it look like the figma version, but also to make it look beautiful and presentable. nice work

yours
Screenshot (741)
figma version
Screenshot (742)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+        </h1>
+
+        <p class="para-section">
+          I&#8217;m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don&#8217;t hestiate to contact me.
+        </p>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] nice work. Please make your work responsive until the desktop screen size
Screenshot (743)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @zeff96 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️ 
Check the comments under the review
tufoin

Optional suggestions
NB - [optional] please do not push commits to the main branch, instead the main branch will be updated when the feature branch has been approved and merged to it. This will help the reviewer see all your changes in the feature branch. nice work

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (zeff96) [reply index.html]: @tufoinnkuo10 Thanks, the issue closed

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @zeff96 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 🏴 
 
-  [ ] Linters are passing 💯 
-  [ ] Right commit messages 🍸 
-  [ ] Proper Gitflow ✔️ 
-  [ ] Great UX design
-  Proper Use of README file

## Optional suggestions

- [ ] I suggest you always use a descriptive branch name for your projects. `Portfolio-Feature-CSS-HTML` sounds a lot more descriptive than `HTML/CSS` which is too genric. Kindly take note of future projects. Well done 👏  

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zeff96** [comment]
@Shedrack-Sunday Thanks for the advise not taken, going forward i will use a descriptive branch name as suggested

**Pull request merged**